src/backend/config.ts

```diff
@@ -323,7 +323,7 @@ class GlobalConfigV0 extends GlobalConfig {
       beforeLaunchScriptPath: '',
       afterLaunchScriptPath: '',
       disableUMU: false,
-      verboseLogs: false
+      verboseLogs: true
```

Comment:
enabled by default, since they can be disabled+play at the same time with a single action now

but by default we can get more logs for new users